myapp/tests/test_views.py

```diff
@@ -0,0 +1,29 @@
+from django.test import TestCase
+
+from myapp.exceptions.stock_service import StockServerUnReachable, StockSymbolNotFound
+from myapp.stock_api import get_stock_info
+
+
+class ViewsTestCase(TestCase):
```

Comment:
Maybe we could name our test case classes according to the feature we are testing (i.e. `StockViewTestCase`). That way, we could group our tests in seperate classes in the same file.

Reply:
I agree with you, I will refactor my code.